uci-pharmsci/README.md

```diff
@@ -9,3 +9,29 @@ This will ultimately contain syllabus information, lecture notes and slides, req
 - `syllabus.md`: Syllabus for PharmSci 175/275
 - `175`: Undergraduate course (PharmSci 175)-specific materials (e.g. assignments)
 - `275`: Graduate course (PharmSci 275)-specific materials (e.g. assignments)
+
+## Getting Started
+- **[How to use Jupyter Notebook](http://jupyter-notebook-beginner-guide.readthedocs.io/en/latest/execute.html)**
+- **[Setup and Installation](getting-started.md)**
+
+## Requirements
+- [Anaconda Python](https://www.continuum.io/downloads): Anaconda is the leading open data science platform powered by Python.
+- [VMD](http://www.ks.uiuc.edu/Development/Download/download.cgi?PackageName=VMD): VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular systems using 3-D graphics and built-in scripting.
+  - Recommended Tutorials:
+    - [Introduction](http://www.ks.uiuc.edu/Training/Tutorials/vmd/tutorial-html/node1.html)
+    - [Working with a Single Molecule](http://www.ks.uiuc.edu/Training/Tutorials/vmd/tutorial-html/node2.html)
+    - [Trajectories and Movie Making](http://www.ks.uiuc.edu/Training/Tutorials/vmd/tutorial-html/node3.html)
+- [OpenEye-toolkits](https://www.eyesopen.com/toolkit-development)
+  - [Documentation](https://docs.eyesopen.com/toolkits/python/index.html)
+- [XQuartz](https://www.xquartz.org/) - X11 for MacOS
```

Comment:
Do you know what requires XQuartz? This would primarily be for, say, using GreenPlanet, right?